docs/book/src/reference/labels_and_annotations.md

```diff
@@ -19,38 +19,39 @@
+| cluster.x-k8s.io/skip-remediation                                 | It is used to mark the machines that should not be considered for remediation by MachineHealthCheck reconciler.                                                                                                                                                                                                                                                                                                                                                                                                                                             |
+| cluster.x-k8s.io/managed-by                                       | It can be applied to InfraCluster resources to signify that some external system is managing the cluster infrastructure. Provider InfraCluster controllers will ignore resources with this annotation. An external controller must fulfill the contract of the InfraCluster resource. External infrastructure providers should ensure that the annotation, once set, cannot be removed.                                                                                                                                                                     |
+| cluster.x-k8s.io/replicas-managed-by                              | It can be applied to MachinePool resources to signify that some external system is managing infrastructure scaling for that pool. See [the MachinePool documentation](../developer/architecture/controllers/machine-pool.md#externally-managed-autoscaler) for more details.                                                                                                                                                                                                                                                                                |
+| cluster.x-k8s.io/skip-machineset-preflight-checks                 | It can be applied on MachineSet resources to specify the list of pre-flight checks to skip during MachineSet reconciliation. The value is a coma seperated list of skippable pre-flight checks. Supported pre-flight checks are: All, KubeadmVersion, KubernetesVersion, ControlPlaneStable.                                                                                                                                                                                                                                                                |
```

Comment:
```suggestion
| cluster.x-k8s.io/skip-machineset-preflight-checks                 | It can be applied on MachineDeployment and MachineSet resources to specify a comma-separated list of preflight checks that should be skipped during MachineSet reconciliation. Supported pre-flight checks are: All, KubeadmVersion, KubernetesVersion, ControlPlaneStable.                                                                                                                                                                                                                                                                |
```